ecmd-core/capi/ecmdClientCapi.C

```diff
@@ -24,6 +24,7 @@
 //----------------------------------------------------------------------
 //  Includes
 //----------------------------------------------------------------------
+#include <fstream>
```

Comment:
This isn't needed.  The function that required this include was dropped for ecmd 15.  

You could resubmit with dropping the slibclean from this file though as you did in master.